switchbot/devices/lock.py

```diff
@@ -26,6 +26,7 @@
 COMMAND_GET_CK_IV = f"{COMMAND_HEADER}0f2103"
 COMMAND_LOCK_INFO = f"{COMMAND_HEADER}0f4f8101"
 COMMAND_UNLOCK = f"{COMMAND_HEADER}0f4e01011080"
+COMMAND_NOT_FULLY_UNLOCK = f"{COMMAND_HEADER}0f4e010110a0"
```

Comment:
This seems like it should be called unlatch. Not fully unlock doesn't really explain what's going on here.

I wouldn't expect someone to ask me to not fully unlock a door

Reply:
Another way to think about it would be: what would you ask a voice assistant to do to execute this action

Reply:
I am not native English speaker. But it seems weird to me call this command unlatch if this just unlock door... not hold the latch. I don't know why they implemented it this way. Unlock command holds the latch if feature is enabled. It would make more sense if they make new command for unlatching not for unlocking without holding the latch.

Reply:
I'm not 100% sure what the new command does as I don't have the lock, but maybe `COMMAND_UNLOCK_WITHOUT_UNLATCH`

Reply:
Renaming unlock to open and not_fully_unlock to unlock would be nice. But it would break backwards compatibility so... its not probably the best idea.

Reply:
maybe add `open` and `unlock_and_open`

Reply:
I don't know about that. That seems misleading too to me. The name NOT_FULLY_UNLOCK is at least some how consistent with name of the state that the lock return NOT_FULLY_LOCKED.

Reply:
Lets call it `unlock_without_unlatch`